avocado/plugins/spawners/podman.py

```diff
@@ -176,6 +186,16 @@ async def spawn_task(self, runtime_task):
 
         return returncode == 0
 
+    async def create_task_output_dir(self, runtime_task):
+        task = runtime_task.task
+        output_dir_path = os.path.join(self.job_output_dir,
+                                       task.identifier.str_filesystem)
+        output_podman_path = os.path.join('/tmp', 'avocado-results')
```

Comment:
I think that there shouldn't be any confusion with other runners because, this is the dir inside container which is new for each task. But definitely the `~/avocado/job-results/spawners/podman/` is better, and I will use it.